packages/block-editor/src/components/block-list/content.scss

```diff
@@ -377,18 +377,24 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	z-index: z-index("{core/image aligned left or right} .wp-block");
 }
 
-body.is-zoomed-out {
-	display: flex;
-	flex-direction: column;
+iframe[name="editor-canvas"].edit-site-visual-editor__editor-canvas.is-zoomed-out {
+	background: #ddd;
+}
 
-	> .is-root-container {
-		flex: 1;
+.edit-site-visual-editor__editor-canvas.is-zoomed-out {
+	body {
```

Comment:
I don't think you select the body element from outside the iframe? `edit-site-visual-editor__editor-canvas` is outside and `body` inside, so this doesn't work I think.

Reply:
edit-site-visual-editor__editor-canvas is a class on the iframe. I moved the is-zoomed-out class from the body to the iframe to fix the issue in safari. The selector needed to be more specific than just iframe.is-zoomed-out, hence why I also used the edit-site-visual-editor__editor-canvas class too.

Reply:
You can't target `body` from outside the iframe though? I don't think `.edit-site-visual-editor__editor-canvas.is-zoomed-out body` will work.

Reply:
oh yeah, you are right! ok, I'll go think more about this